lib/wrappers/genesisProtocol.ts

```diff
@@ -1177,50 +1178,65 @@ export interface GenesisProtocolParams extends TxGeneratingFunctionOptions {
    */
   minimumStakingFee: BigNumber | string;
   /**
-   * The time limit in seconds for a proposal to be in absolute voting mode.
+   * The time limit in seconds that a proposal can be in the preBoosted phase before
+   * it will be automatically closed, in seconds, with a winning vote of NO, regardless
+   * of the actual value of the winning vote at the time expiration.
+   * Note an attempt must be made to execute before the proposal state will actually change.
    * Default is 1814400 (three weeks).
    */
   preBoostedVotePeriodLimit: number;
   /**
-   * The percentage of the absolute vote that must be exceeded to result in a win.
-   * Must be between 0 and 100.
+   * The percent of the DAO's total supply of reputation that, when exceeded
+   * by the amount of reputation behind a vote (yes or no), will result
+   * in the immediate execution of the proposal, during either the preboosted
+   * or boosted phases. Must be between 0 and 100.
```

Comment:
0<preBoostedVotePeriodLimit <=100

Reply:
@orenyodfat There is currently no validation on preBoostedVotePeriodLimit in Arc.  I'm not sure where I got that statement about the required range of values.

Reply:
@orenyodfat Are you sure you aren't referring to preBoostedVoteRequiredPercentage?

Reply:
yea .was referring to preBoostedVoteRequiredPercentage .it is ok. 👍